### PR TITLE
Remove @return from docblock.

### PR DESCRIPTION
**Pull request by cxj**
There is no returned value from this method.

### CODE REVIEW

File: src/Service/ResumeService.php [hunk 1]
```diff
@@ -91,8 +91,6 @@ public function __construct(
      *
      * @param Auth $auth An authentication tracker.
      *
-     * @return bool Whether or not a session still exists.
```
Comment (harikt) [inline src/Service/ResumeService.php:94]: I think we should use `@return null` . See https://github.com/auraphp/Aura.Auth/blob/f83060d3004af3777f0fc2b6c6672174e6133f7d/src/Service/LogoutService.php#L68

Comment (cxj) [reply src/Service/ResumeService.php:94]: Ok, can do that.

Comment (cxj) [reply src/Service/ResumeService.php:94]: See PR #87.

**Comment by harikt** [comment]
closing in favour of 87 being merged.

**Pull request closed without merging**